overcast/runner/__init__.py

```diff
@@ -128,6 +128,83 @@ def get_creds_from_env():
 #    d['cacert'] = os.environ.get('OS_CACERT', None)
     return d
 
+
+class node(object):
+    dr = self.DeploymentRunner()
+
+    def __init__(self, name, info, keypair, userdata, mappings=None, record_resource=None):
+        self.record_resource = lambda *args, **kwargs: None
+        self.info     = info
+        self.networks = dr.networks or {}
```

Comment:
When you have self.runner, you don't need to keep a copy of it here, just use it directly from self.runner.networks.